.github/workflows/ci-cd.yml

```diff
@@ -367,6 +367,11 @@ jobs:
         with:
           bundler-cache: true
       - run: bundle exec rails db:test:prepare
+      - name: Cache cypress binary
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/Cypress
+          key: ${{ runner.os }}-cypress-binary-${{ hashFiles('**/yarn.lock') }}
```

Comment:
We have been experiencing a flaky spec that can happen when Cypress fail to download. Caching the binary will remove the need to download it every time from source.